src/components/LookupButton.tsx

```diff
@@ -1,23 +1,21 @@
-// @flow
-'use strict'
-
 import * as React from 'react'
 import clsx from 'clsx'
 import useLookupNotification from '../hooks/useLookupNotification'
 import { lookupValidationMessage } from '../services/form-validation'
 
-/* ::
 type Props = {
-  value: mixed | void,
-  validationMessage: string | void,
-  hasMarginTop?: boolean,
-  isInputButton?: boolean,
+  value: unknown | void
```

Comment:
```suggestion
  value?: unknown
```

Reply:
I would prefer not to make this props optional, and would rather them be required so that devs do not mistakenly omit the prop when it actually should be passed a value.